rust-toolchain.toml

```diff
@@ -1,3 +1,3 @@
 [toolchain]
```

Comment:
Also commented on Slack: could we split this file into one per package? That way we can update some of them independently. E.g. I think `c2rust-analyze` and `c2rust-pdg` could be on stable Rust.

Reply:
`c2rust-analyze` and `c2rust-pdg` use rustc internals, so they need nightly.

Reply:
For `c2rust-transpile` and it's dependencies, those can build on stable, but `c2rust`, if it depends on things like `c2rust-analyze` (it doesn't currently, but that was the future intention), I'm not sure how to have `rust-toolchsin.toml` work there, as we want it to build on both nightly and stable. 

This is also an issue for `rav1d`, so figuring out how to do this would be nice.